app/mailers/user_mailer.rb

```diff
@@ -377,6 +377,18 @@ def admin_deleted_work_notification(user, work)
     end
   end
 
+  def admin_hidden_series_notification(creation_id, user_id)
+    @user = User.find_by(id: user_id)
+    @series = Series.find_by(id: creation_id)
+
+    I18n.with_locale(@user.preference.locale.iso) do
+      mail(
+        to: @user.email,
+        subject: default_i18n_subject(app_name: ArchiveConfig.APP_SHORT_NAME)
```

Comment:
This needs a comment to help out i18n-tasks:
```suggestion
        # i18n-tasks-use t('user_mailer.admin_hidden_series_notification.subject')
        subject: default_i18n_subject(app_name: ArchiveConfig.APP_SHORT_NAME)
```

Reply:
Update: The comment is no longer needed! Just merge the current master branch into this PR branch and i18n-tasks will recognize `default_i18n_subject` automatically.